src/webui/www/private/scripts/speedslider.js

```diff
@@ -1,18 +1,3 @@
-/*
-
-Script: Parametrics.js
-	Initializes the GUI property sliders.
-
-Copyright:
-	Copyright (c) 2007-2008 Greg Houston, <http://greghoustondesign.com/>.
-
-License:
-	MIT-style license.
-
-Requires:
-	Core.js, Window.js
-
-*/
 MochaUI.extend({
```

Comment:
I would add `'use strict';` if there isn't much trouble.
Also would add the GPL license header too.

Reply:
Added the GPL header and `use strict`. Didn't even have to make any js changes to conform to strict mode.